test/prospectScraperTest/MockDraftPickTests.cs

```diff
@@ -27,6 +27,14 @@ public void ConvertPickToRound_Returns_Correct_Round(string pick, int expectedRo
             Assert.Equal(expectedRoundValue, MockDraftPick.ConvertPickToRound(pick));
         }
 
+        [Theory]
+        [InlineData("", 0)]
+        [InlineData("-33", 0)]
+        public void ConvertPickToRound_Returns_Zero(string pick, int expectedRoundValue)
```

Comment:
This will sound like a weird request, but when setting up the test parameters, if we are going to call the params expectedRoundValue, ...pick, we should try to make the test method signature match. This helps in readability. Likewise, should we updated the amount of values in the inline data to be more, so we can test many rounds (and perhaps set a default of int expectedRoundValue = 0 so we don't always have to pass it)?

Reply:
literally just the other way around:
`        [InlineData("")]
        [InlineData("-33")]
        public void ConvertPickToRound_Returns_Zero(string pick, int expectedRoundValue = 0)`
We would need to add a set of inline data that includes an example where we don't have an expectedRoundValue of more than 0, however.

Likewise, the other thing I was saying, was that we call "expectedRoundValue" second in the method signature, but it is the first parameter used. so you could flip the parameters in the method signature. 

Lastly, the test name ConvertPickToRound_Returns_Zero
We could make this clearer by having a standard. Why is it _Returns_Zero? Why not ConvertPickToRound_ReturnsZero? Or ConvertPickToRoundReturnsZero? I personally think the second one, ConvertPickToRound_ReturnsZero, is correct, as it names the action and the expected results. However, I don't like the fact that we are just getting a return of zero. We should test all use cases of the underlying method so that we don't just do happy path testing.

Reply:
I'm going to merge this.  Feel free to create an issue if you want to, though.